ansible/README.md

```diff
@@ -0,0 +1,157 @@
+# Collector Ansible Tools
+
+The files contained within this directory are [ansible](https://www.ansible.com/resources/get-started)
+roles and playbooks written to simplify VM life-cycle control, and execution of testing, both
+during development and on CI systems[^1].
+
```

Comment:
Include prerequisites section. Should specify that ansible and requests library should be installed. Possibly others.